omod/src/main/webapp/pages/allergy.gsp

```diff
@@ -60,11 +60,11 @@ ${ ui.includeFragment("allergyui", "removeAllergyDialog") }
             <div id="allergens">
 
                 <div id="types" class="button-group horizontal">
-                    <% allergenTypes.each { category -> %>
+                    <% allergenTypes.each { category ->  if (category.name()!="ENVIRONMENT") { %>
```

Comment:
@dkayiwa This is done to comply with what Darius commented here :  https://github.com/openmrs/openmrs-module-allergyui/pull/26 . Basically this comparison ensures that a seperate tab for environment is not created in the UI although OpenMRS core supports it.